lib/diagnostic.js

```diff
@@ -14,9 +14,24 @@
 'use strict';
 
 const fs = require('fs'); // jshint ignore: line
-
+let inFile = filename;
 const sumLines = (filename, callback) => {
-  /* your response here */
+
+  fs.readFile(inFile, 'utf-8', (error, data) => {
+  if (error) {
+    console.error(error.stack);
+    return;
+  }
+
+  let dataArray = data.split('\n');
+
+
+
+  dataArray.reduce((a, b) a + b )=>{
+    console.log();
```

Comment:
Yes yes yes! But! Look at the directions again: "[This function] should add all the numbers and pass the result as the second argument to the callback provided. e.g `callback(null, sum);`."